ci-operator/config/openshift/ocs-operator/openshift-ocs-operator-master.yaml

```diff
@@ -20,38 +27,88 @@ images:
       - destination_dir: .
         source_path: /go/src/github.com/openshift/ocs-operator/build/_output/bin/ocs-operator
   to: ocs-operator
+
 - dockerfile_path: openshift-ci/Dockerfile.registry.intermediate
   from: operator-registry
   to: operator-registry-base
+
 - dockerfile_path: openshift-ci/Dockerfile.registry.build
   from: operator-registry-base
   to: ocs-registry
+
+- dockerfile_path: build/Dockerfile
+  from: os
+  inputs:
+    bin:
+      paths:
+      - destination_dir: _output/bin
+        source_path: /go/src/github.com/openshift/ocs-operator/build/_output/bin/ocs-operator
+  to: ocs-operator-nightly
+  optional: true
+
+- dockerfile_path: deploy/Dockerfile
+  from: operator-registry
+  inputs:
+    src:
+      paths:
+      - destination_dir: deploy
+        source_path: /go/src/github.com/openshift/ocs-operator/deploy/Dockerfile
+      - destination_dir: .
+        source_path: /go/src/github.com/openshift/ocs-operator/deploy/olm-catalog
+  to: ocs-registry-nightly
+  optional: true
+
+- dockerfile_path: must-gather/Dockerfile
+  from: must-gather
+  inputs:
+    src:
+      paths:
+      - destination_dir: must-gather
+        source_path: /go/src/github.com/openshift/ocs-operator/must-gather/Dockerfile
+      - destination_dir: .
+        source_path: /go/src/github.com/openshift/ocs-operator/must-gather/Centos-Base.repo
+      - destination_dir: .
+        source_path: /go/src/github.com/openshift/ocs-operator/must-gather/collection-scripts
+      - destination_dir: .
+        source_path: /go/src/github.com/openshift/ocs-operator/must-gather/templates
+  to: ocs-must-gather-nightly
+  optional: true
+
 promotion:
   additional_images:
     ocs-operator-tests: src
+  excluded_images:
+  - ocs-operator
+  - operator-registry-base
```

Comment:
For `excluded_images`, some files are using `operator-registry-base` and some are using `ocs-registry-base`. Is this intentional? The former looks like it'd be more correct.

Reply:
You're right, I forgot to update the configs for the other branches after I fixed the master config.

Reply:
Fixed in #5407